functions/src/api/cards/api/postBuyCards.ts

```diff
@@ -51,7 +51,7 @@ export const postBuyCards: RequestHandler<{}, {}, PostBody> = async (
 
   try {
     const updatedPlayer = await buyCards(sessionId, playerId, amount);
-    res.send(updatedPlayer);
+    res.send({ player: updatedPlayer });
```

Comment:
Why this change?

Reply:
This change was made considering the scenario where the client would need the player and the card popped.
But this is not required as the existing listeners in the FE are dope 😉  

undoing...